src/webapp/src/auth/auth.service.ts

```diff
@@ -388,7 +393,8 @@ export class AuthService {
       email: data.email,
       email_verified: data.email_verified,
       subject: data.sub,
-      extra: data
+      extra: data,
+      tokens: { access_token: req.body.access_token }
```

Comment:
Let's change the api of this function like the other two, passing accessToken and refreshToken from outside. And drop `req` at this level.